packages/pancake-uikit/src/widgets/Menu/Menu.tsx

```diff
@@ -50,30 +63,8 @@ const Inner = styled.div<{ isPushed: boolean; showMenu: boolean }>`
   }
 `;
 
-const MobileOnlyOverlay = styled(Overlay)`
-  position: fixed;
-  height: 100%;
-
-  ${({ theme }) => theme.mediaQueries.nav} {
-    display: none;
-  }
-`;
-
-const Menu: React.FC<NavProps> = ({
-  userMenu,
-  globalMenu,
-  isDark,
-  toggleTheme,
-  langs,
-  setLang,
-  currentLang,
-  cakePriceUsd,
-  links,
-  children,
-}) => {
-  const { isMobile, isTablet } = useMatchBreakpoints();
-  const isSmallerScreen = isMobile || isTablet;
-  const [isPushed, setIsPushed] = useState(!isSmallerScreen);
+const Menu: React.FC<NavProps> = ({ userMenu, globalMenu, isDark, cakePriceUsd, links, subLinks, children }) => {
```

Comment:
I'm not too familiar with the menu code or spec... but do we still want the hide-on-scroll behaviour on lns 71-98?  https://github.com/pancakeswap/pancake-toolkit/pull/261/files#diff-059db1db1b94417eed58e3127ec02a05c8bf0a76819430a59a26c6179bcb9b04R71-R98

Reply:
Good question, I think so but I'll double check.